drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -411,9 +411,12 @@ function DragAndDropTemplates(configuration) {
             ];
             popup_content = h("div.popup-content", {}, have_messages ? content_items : []);
         } else {
-            popup_content = h("div.popup-content", {}, msgs.map(function(message) {
-                return h("p", {innerHTML: message.message});
-            }))
+            popup_content = h(ctx.last_action_correct ? "div.popup-content" : "div.popup-content.popup-content-incorrect",
+                              {},
+                              msgs.map(function(message) {
+                                return h("span", {innerHTML: message.message});
```

Comment:
Why was this changed from `p` to `span`? `p` makes more sense semantically.

Reply:
`p` element seems to have a style defined saying `text-align:left`. So the popup text is always aligned left. I was not sure how to override the style for the `p` element just for this case.

Reply:
You can override the styles of `p` elements inside the popup like this:

```css
@media screen and (max-width: 480px) {
    .xblock--drag-and-drop .popup .popup-content p {
        text-align: center;
    }
}
```